src/main/scala/edu/berkeley/cs/rise/opaque/Utils.scala

```diff
@@ -301,11 +303,20 @@ object Utils extends Logging {
     val enableSharedKey = sqlContext.getConf("spark.opaque.testing.enableSharedKey", "false")
```

Comment:
Shouldn't the default value be true, in case the user forgets to set it?

Reply:
It depends on what we want the behavior to be. If the code should always default to using a fake key, then yes -- but this means that exceptions such as "cannot encrypt/decrypt without sharedKey" will never occur. I think it makes sense to differentiate between testing case and actual in-practice case, which is why I defaulted to false. The user should always either be explicitly running tests or supplying their own key; anything else would result in error.

Reply:
Oops, yes, you're right -- for I thought enabling shared key means that the user needs to provide it. What about renaming it to `enableTestingKey`, or `enableDummySharedKey`?